Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPIPermissionsCocoa.mm

```diff
@@ -54,19 +54,16 @@
     }, extensionContext().identifier().toUInt64());
 }
 
-void WebExtensionAPIPermissions::contains(NSDictionary *details, Ref<WebExtensionCallbackHandler>&& callback, NSString **)
+void WebExtensionAPIPermissions::contains(NSDictionary *details, Ref<WebExtensionCallbackHandler>&& callback, NSString **outExceptionString)
 {
     RELEASE_LOG(Extensions, "permissions.contains()");
 
     HashSet<String> permissions, origins;
     WebExtension::MatchPatternSet matchPatterns;
     parseDetailsDictionary(details, permissions, origins);
 
-    NSString *errorMessage;
-    if (!validatePermissionsDetails(permissions, origins, matchPatterns, @"contains()", &errorMessage)) {
-        callback->reportError(errorMessage);
+    if (!validatePermissionsDetails(permissions, origins, matchPatterns, @"permissions.contains()", outExceptionString))
```

Comment:
`contains()` doesn't set the error on the callback, only `remove()` and `request()`.